python/TestHarness/RunParallel.py

```diff
@@ -154,7 +154,7 @@ def getOutputFromFiles(self, tester):
       # Obtain path and append processor id to redirect_output filename
       file_path = os.path.join(tester.specs['test_dir'], tester.name() + '.processor.{}'.format(processor_id))
       with open(file_path, 'r') as f:
-        file_output += self.readOutput(f)
+        file_output += "#"*80 + "\nOutput from processor " + str(processor_id) + "\n" + "#"*80 + "\n" + self.readOutput(f)
```

Comment:
I'm just going to trust you that this is what you want here :-)  The rest looks good.